givitsite/givitsite/settings.py

```diff
@@ -128,5 +131,5 @@
 STATIC_URL = '/static/'
 
 STATICFILES_DIRS = [
-    os.path.join(BASE_DIR,'static')
+    os.path.join(BASE_DIR, 'static')
```

Comment:
generally you should keep stylistic fixed out of PRs that include functionality.

Reply:
yes i saw now, its mistake.